### PR TITLE
Consistent aragon branding

### PR DESCRIPTION
**Pull request by brickpop**
- Removing NextJS logos
- Using neutral Aragon logos and favicons
- Cleaner default styling

image

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,30 +1,44 @@
...
         <p className="text-lg">Get in touch with us for more information.</p>
-      </section>
-
+      </Section>
     </main>
-  )
+  );
 }
+
+const Section = ({ children }: { children: ReactNode }) => {
```
Comment (carlosgj94) [inline app/page.tsx:38]: dope

File: public/logo-bw-lg.png
Comment (carlosgj94) [inline public/logo-bw-lg.png:1]: I didn't even know we had these awesome eagles!

File: app/_app.tsx
Comment (carlosgj94) [inline app/_app.tsx:1]: Good idea. 
I wonder if this gets in conflict with the javascript way to generate the other metadata. We won't be modifying the favicons tho, so we good for now.

Link to the docs for future reference: https://nextjs.org/docs/app/api-reference/functions/generate-metadata

**Review by carlosgj94 — APPROVED** [review]

File: app/_app.tsx
Comment (brickpop) [reply app/_app.tsx:1]: The `NextScript` tag takes charge of loading its own scripts. If we want to manually load other stuff, this is the place to go

**Pull request merged**